Tests/Foundation/Tests/TestAffineTransform.swift

```diff
@@ -7,383 +7,741 @@
+    #if canImport(SwiftFoundation) && !DEPLOYMENT_RUNTIME_OBJC
+        @testable import SwiftFoundation
+    #else
+        @testable import Foundation
+    #endif
+#endif
+
+// MARK: - Tests
+
+class TestAffineTransform: XCTestCase {
     private let accuracyThreshold = 0.001
 
     static var allTests: [(String, (TestAffineTransform) -> () throws -> Void)] {
         return [
-            ("test_BasicConstruction", test_BasicConstruction),
-            ("test_IdentityTransformation", test_IdentityTransformation),
-            ("test_Scale", test_Scale),
-            ("test_Scaling", test_Scaling),
-            ("test_TranslationScaling", test_TranslationScaling),
-            ("test_ScalingTranslation", test_ScalingTranslation),
-            ("test_Rotation_Degrees", test_Rotation_Degrees),
-            ("test_Rotation_Radians", test_Rotation_Radians),
-            ("test_Inversion", test_Inversion),
-            ("test_IdentityTransformation", test_IdentityTransformation),
-            ("test_Translation", test_Translation),
-            ("test_TranslationComposed", test_TranslationComposed),
-            ("test_AppendTransform", test_AppendTransform),
-            ("test_PrependTransform", test_PrependTransform),
-            ("test_TransformComposition", test_TransformComposition),
-            ("test_hashing", test_hashing),
-            ("test_rotation_compose", test_rotation_compose),
-            ("test_translation_and_rotation", test_translation_and_rotation),
-            ("test_Equal", test_Equal),
-            ("test_NSCoding", test_NSCoding),
+            ("testConstruction", testConstruction),
+            ("testBridging", testBridging),
+            ("testEqualityHashing", testEqualityHashing),
+            ("testVectorTransformations", testVectorTransformations),
+            ("testIdentityConstruction", testIdentityConstruction),
+            ("testIdentity", testIdentity),
+            ("testTranslationConstruction", testTranslationConstruction),
+            ("testTranslation", testTranslation),
+            ("testScalingConstruction", testScalingConstruction),
+            ("testScaling", testScaling),
+            ("testRotationConstruction", testRotationConstruction),
+            ("testRotation", testRotation),
+            ("testTranslationScaling", testTranslationScaling),
+            ("testTranslationRotation", testTranslationRotation),
+            ("testScalingRotation", testScalingRotation),
+            ("testInversion", testInversion),
+            ("testPrependTransform", testPrependTransform),
+            ("testAppendTransform", testAppendTransform),
+            ("testNSCoding", testNSCoding),
         ]
     }
-    
-    func checkPointTransformation(_ transform: NSAffineTransform, point: NSPoint, expectedPoint: NSPoint, _ message: String = "", file: StaticString = #file, line: UInt = #line) {
+}
+ 
+// MARK: - Helper
+
+extension TestAffineTransform {
+    func assert(
```

Comment:
I would suggest a different name to `assert()` as that is common global function, maybe just `checkValue` or so.